app.server.js

```diff
@@ -37,6 +39,19 @@ app.use(
 // FIX ME: here we have to get the google APIkey in another way.
 app.locals.googleCredential = new GoogleCredentialController(GOOGLE_CREDENTIAL);
 
+app.use((req, res, next) => {
+  var proto = req.connection.encrypted ? 'https' : 'http';
+  // only do this if you trust the proxy
+  proto = req.headers['x-forwarded-proto'] || proto;
+
+  if (proto !== PRODUCTION_PROTOCOL){
+      res.redirect(`${PRODUCTION_PROTOCOL}://${req.header('host')}${req.url}`)
+  }else{
+    next()
+  }
+})
```

Comment:
Suggestion:
I think instead pass a protocol env, users should define a `ENFORCE_SSL` variable:

 ```
if (req.secure) {
    return next()
}

if (process.env.ENFORCE_SSL){
     return res.redirect(`${PRODUCTION_PROTOCOL}://${req.hostname}${req.url}`)
}

next();

Reply:
I updated with your suggestion :)